prow/scripts/governance.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+set -e
+
+readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck disable=SC1090
+source "${SCRIPT_DIR}/library.sh"
+
+readonly MILV_IMAGE="magicmatatjahu/milv:0.0.6"
+OUTPUT=0
+
+# read arguments
+while test $# -gt 0; do
+    case "$1" in
+        --repository)
+            shift
+            readonly REPOSITORY_NAME=$1
+            shift
+            ;;
+        --repository-dir)
+            shift
+            readonly REPOSITORY_DIR=$1
+            shift
+            ;;
+        --full-validation)
+            shift
+            readonly FULL_VALIDATION=$1
+            shift
+            ;;
+        *)
+            echo "$1 is not a recognized flag!"
+            exit 1;
+            ;;
+    esac
+done
+
+if [[ -z "${REPOSITORY_NAME}" ]]; then
```

Comment:
How about extracting it from `$REPOSITORY_DIR`?

Reply:
I think that extracting `REPOSITORY_NAME` from `REPOSITORY_DIR` is a bad idea, because I would like to preserve the generic of this script so that TC can also fire it on their computers.